.github/scripts/test_db_sync.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
rename file to indicate this is for testnet.

Reply:
Renamed.